Formula/dps8m.rb

```diff
@@ -0,0 +1,31 @@
+class Dps8m < Formula
+  desc "Simulator for the Multics dps-8/m mainframe"
+  homepage "http://ringzero.wikidot.com"
+  url "https://downloads.sourceforge.net/project/dps8m/Release%201.0/source.tgz"
+  version "1.0"
+  sha256 "51088dd91de888b918644c431eec22318640d28eb3050d9c01cd072aa7cca3c7"
+
+  depends_on "libuv"
+
+  def install
+    bin.mkpath
+    system "make", "INSTALL_ROOT=#{prefix}", "install"
+  end
+
+  test do
+    (testpath/"test.exp").write <<-EOS.unindent.undent
+      spawn #{bin}/dps8
+      expect "DPS8/M emulator (git b7a50ffc)
+              Production build
+              DPS8M system session id is 70486
+              Please register your system at https://ringzero.wikidot.com/wiki:register
+              or create the file 'serial.txt' containing the line 'sn: 0'.
+              Couldn't open Devices.txt
+              Unknown command: fnpload Devices.txt
+              FNP telnet server port set to 6180
+
+              DPS8M simulator V4.0-0 Beta        git commit id: c420925a"
```

Comment:
Please remove the version and git commit id from the pattern matching. It will make future updates easier.